lms/views/admin/email.py

```diff
@@ -76,3 +78,35 @@ def post(self):
         )
 
         return HTTPFound(location=self.request.route_url("admin.email"))
+
+    @view_config(
+        route_name="admin.email.preview.instructor_email_digest",
+        request_method="GET",
+        renderer="lms:templates/email/instructor_email_digest/body.html.jinja2",
+    )
+    def preview_instructor_email_digest(self):
+        return INSTRUCTOR_EMAIL_DIGEST_TEMPLATE_VARS
+
+
+#: Test template variables that the admin page will pass to the instructor_email_digest templates.
+INSTRUCTOR_EMAIL_DIGEST_TEMPLATE_VARS = {
```

Comment:
We could eventually take these from the admin page if the template itself gets more complicated, having them hardcoded here is more than enough for now :+1: